e2e-tests/yarn.lock

```diff
@@ -2678,4 +2678,4 @@ yauzl@^2.10.0:
   integrity sha512-p4a9I6X6nu6IhoGmBqAcbJy1mlC4j27vEPZX9F4L4/vZT3Lyq1VkFHw/V/PUcB9Buo+DG3iHkT0x3Qya58zc3g==
   dependencies:
     buffer-crc32 "~0.2.3"
-    fd-slicer "~1.1.0"
+    fd-slicer "~1.1.0"
```

Comment:
The file should not be tracked here.
How about 'git checkout main e2e-tests/yarn.lock' and commit the file again? then the file will not be shown here.